lesson-34/index.ts

```diff
@@ -0,0 +1,100 @@
+interface IFaculties {
+    id: number;
+    faculty: string;
+    subjects: string[];
+    countStudents: number
+}
+
+
+
+
+1. const faculties = [
+    {
+		id: 1,
+		faculty: "History department",
+		subjects: ["The World History", "History of Rome"]
+		countStudents: 44
+	},
+	{
+		id: 2,
+		faculty: "Department of Biology",
+		subjects: ["biology", "chemistry"],
+		countStudents: 50
+	},
+	{
+		id: 3,
+		faculty: "Faculty of Mathematics",
+		subjects: ["mathematics", "geometry", "trigonometry"],
+		countStudents: 72
+	},
+	{
+		id: 4,
+		faculty: "Faculty of Design",
+		subjects: ["ui", "ux", "graphic design"],
+		countStudents: 37
+	}
+];
+
+interface IMovies {
+        id: number;
+		title: string;
+		year: number;
+		released: string;
+		runtime: string;
+		genre: string[],
+		director: string;
+		writer: string;
+		actors: string[];
+		plot: string;
+		country: string;
+		poster: string;
+		imdbRating: number;
+		imdbVotes: number;
+		type: string;
+		boxOffice: string;
+		production: string,
+}
+
+2.
+const movies = [
```

Comment:
And here you need to define type of this array of movies, like this: 
`const movies: IMovies[] = [...]`